keps/sig-cli/3659-kubectl-apply-prune/README.md

```diff
@@ -528,40 +443,40 @@ We may revisit this and converge on a single, mandatory hint annotation before t
 When a tool that wants to list the members of an ApplySet and cannot determine the list of GKs (i.e. because neither hint annotation is used), it may support “discovery”, likely warning that a full cluster crawl is being attempted. Insufficient permissions errors are likely with such functionality, and when they are encountered, the tool should warn that the membership list may be incomplete.
 
 ```yaml
-applyset.k8s.io/contains-group-kinds: <kind>.<group>[,]` # OPTIONAL
+applyset.kubernetes.io/contains-group-resources: <resource>.<group>[,]` # OPTIONAL
```

Comment:
This is what we actually did in the alpha, but we may want to go back to the original proposal: https://github.com/kubernetes/enhancements/pull/3661#issuecomment-1500284301